drape/support_manager.cpp

```diff
@@ -139,48 +142,32 @@ bool SupportManager::IsVulkanForbidden(std::string const & deviceName,
       Configuration{"Mali-G72", {1, 1, 131}, {26, 0, 0}},
   };
 
-  for (auto const & d : kBannedDevices)
+  return std::any_of(kBannedConfigurations, kBannedConfigurations + ARRAY_SIZE(kBannedConfigurations), [&](auto const & c)
```

Comment:
IMO, before was readable, now is an unreadable long string without any benefits.

Reply:
Returned back. We have different taste.